app/views/users/webauthn_setup/new.html.erb

```diff
@@ -1,6 +1,8 @@
 <% self.title = @presenter.page_title %>
 
-<%= image_tag asset_url(@presenter.image_path), alt: '', width: '90', class: 'margin-left-1 margin-bottom-2', role: 'img' %>
+<% if @platform_authenticator %>
+  <%= image_tag asset_url(@presenter.image_path), alt: '', width: '90', class: 'margin-left-1 margin-bottom-2', role: 'img' %>
+<% end %>
```

Comment:
If we're only rendering an image for platform authenticators, we could remove the condition in `image_path` and remove `security-key.svg` altogether depending if it's unused elsewhere.

https://github.com/18F/identity-idp/blob/eb1d7acb31605dcceb9d4e4a9a14e6cb099e8218/app/presenters/webauthn_setup_presenter.rb#L28-L34

Alternatively, we could inline `platform-authenticator.svg` and remove the method altogether, or update the logic here to render depending if `image_path` returns anything.

Reply:
Removed `security-key.svg`

Reply:
The code above in `webauthn_setup_presenter.rb` still references the removed image, can we remove that code path as well?

https://github.com/18F/identity-idp/blob/851f88d8c8e5b9d1f806547df7991b30f40b1e83/app/presenters/webauthn_setup_presenter.rb#L28-L34